### PR TITLE
[MODULAR] Feign impairment ceiling increase 1 minute -> 10 hours

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Increases the amount of time feign impairment can last to up to 36000 seconds, being ten hours.

## How This Contributes To The Skyrat Roleplay Experience
The one minute limit seems arbitary, you can cancel the impairment whenever you want and keep extending it manually forever.

## Changelog

:cl:
qol: Feign impairment now has a time limit of 10 hours, instead of 1 minute
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
Could just do away with the cap, really. An hour seems just as arbitrary as a minute does.

**Comment by Zonespace27** [comment]
My concern with removing it entirely is that giving people the ability to set a number to anything without a cap has the potential for bad things

**Comment by tf-4** [comment]
> My concern with removing it entirely is that giving people the ability to set a number to anything without a cap has the potential for bad things

well, just put it to something like 5 hours that a round won't reasonably exceed.

**Comment by Zonespace27** [comment]
Done

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -122,8 +122,8 @@
 	var/duration = input(src, "Enter how long you will feign [impairment]. (1 - 60 seconds)", "Duration in seconds", 25) as num|null
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/human.dm]:
```suggestion
	var/duration = input(src, "Enter how long you will feign [impairment].", "Duration in seconds", 25) as num|null
```

**Review by tf-4 — APPROVED** [review]

**Review by tf-4 — CHANGES_REQUESTED** [review]
Suggested a better way on discord

**Review by tf-4 — APPROVED** [review]

**Comment by hugmeorelse** [comment]
Is there a way to turn it on and off instead? @Zonespace27

**Comment by Zonespace27** [comment]
> Is there a way to turn it on and off instead? @Zonespace27

The variables this affects aren't booleans, and I've said my piece on setting this to be infinite

**Comment by hugmeorelse** [comment]
> > Is there a way to turn it on and off instead? @Zonespace27
> 
> The variables this affects aren't booleans, and I've said my piece on setting this to be infinite

But is there a way to turn on and off, i.e. changing the timer from an hour to one second to make it go away?

**Comment by Zonespace27** [comment]
> > > Is there a way to turn it on and off instead? @Zonespace27
> > 
> > 
> > The variables this affects aren't booleans, and I've said my piece on setting this to be infinite
> 
> But is there a way to turn on and off, i.e. changing the timer from an hour to one second to make it go away?

Yes, this was a function beforehand to let you turn it off.

**Pull request merged**